platforms/pf_test/platform.cpp

```diff
@@ -25,10 +27,33 @@
 static cogip::uartpb::UartProtobuf uartpb(UART_DEV(1));
 
 /* Thread stacks */
-char countdown_thread_stack[THREAD_STACKSIZE_DEFAULT];
+static char heartbeat_thread_stack[THREAD_STACKSIZE_DEFAULT];
 
 static bool copilot_connected = false;
 
+static void *_heartbeat_thread(void *args)
+{
+    (void)args;
+
+    // Init loop iteration start time
+    ztimer_now_t loop_start_time = ztimer_now(ZTIMER_MSEC);
+
+    while (true) {
+        gpio_set(HEARTBEAT_LED);
+        ztimer_sleep(ZTIMER_MSEC, 200);
+        gpio_clear(HEARTBEAT_LED);
+        ztimer_sleep(ZTIMER_MSEC, 200);
+        gpio_set(HEARTBEAT_LED);
+        ztimer_sleep(ZTIMER_MSEC, 200);
+        gpio_clear(HEARTBEAT_LED);
+
+        // Wait thread period to end
+        ztimer_periodic_wakeup(ZTIMER_MSEC, &loop_start_time, 1800);
```

Comment:
No define for thread period?

Reply:
:fearful: done

Reply:
Yes, but you used `static uint16_t` instead of `constexpr`.

Reply:
done